backend/typescript/testUtils/setup.ts

```diff
@@ -0,0 +1,4 @@
+import dotenv from "dotenv";
+import * as path from "path";
+
+dotenv.config({ path: path.resolve(__dirname, "../../../.env") });
```

Comment:
Might not be necessary if running through docker, the env variables have already been set in the container